source/manual/business-readiness-publish-changes.html.md

```diff
@@ -31,23 +31,24 @@ The content item for the finder is updated from a [YAML file][govuk-app-deployme
 
 ## Updating the business readiness finder
 
-The content item for the business readiness finder and its email signup page are published by `search-api`. However the actual config is read in from [govuk-app-deployment-secrets][govuk-app-deployment-secrets].
+The content item for the business readiness finder and its email signup page are published by `search-api`. However the actual config is read in from [govuk-app-deployment-secrets][govuk-app-deployment-secrets], and config for the facet group is managed in [content-tagger][content-tagger].
 
-1. Update [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
-2. Merge the changes to [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
-3. Re-deploy `search-api` so it can pick up the changes in [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
-
-  >> If you have made changes to the facets, you may also need to re-deploy email-alert-api and finder-frontend
-
-4. Run the [`publishing_api:publish_eu_exit_business_finder` rake task][staging-rake-task] in `search-api` to publish the changes. If the finder config has the facets defined in a separate `facet_group` content item and the `links` key in the config contains an entry of `facet_group` with a content id, the task is [`publishing_api:publish_facet_group_eu_exit_business_finder` rake task][staging-rake-task-facet-group]:
+1. Update the facet group in [content-tagger][content-tagger]
+2. Merge the changes to [content-tagger][content-tagger]
+3. Deploy content-tagger
+4. [Import and Publish the facet group](/manual/publishing-a-facet-group.html#how-to-import-or-update-a-facet-group)
+5. Update [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
+6. Merge the changes to [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
+7. Re-deploy `search-api` so it can pick up the changes in [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
+8. Run the [`publishing_api:publish_facet_group_eu_exit_business_finder` rake task][staging-rake-task-facet-group] in `search-api` to publish the changes:
 
     ![download](images/publish-business-readiness.png)
```

Comment:
The image shows a different rake task to the text.  Could the screenshot be updated?

Reply:
I thought I had updated it. I'll double-check.

Reply:
It could be down to the way GitHub is showing the diff

Reply:
You can see the change if you view the file: https://github.com/alphagov/govuk-developer-docs/blob/2aa3075e8222f8c5edd9e69cafba57e258be427d/source/manual/images/publish-business-readiness.png

Reply:
All down to Github then...